extensions/tailscale/src/exit.tsx

```diff
@@ -80,6 +61,25 @@ function ExitNodeList() {
 
   return (
     <List isLoading={exitNodes === undefined}>
+      {isActive && (
+        <List.Item
+          key="_disable"
+          title="Turn off exit node"
+          icon={{
```

Comment:
I would prefer if you removed the icon from this `List.Item` altogether rather than introduce a red dot icon; the dot indicators are for showing the online/offline status of nodes. We don't currently use a red dot in any of our other UIs.

Reply:
Perhaps use the `XMarkCircle` icon instead then? https://developers.raycast.com/api-reference/user-interface/icons-and-images

Reply:
Resolved in 3070902 -- it doesn't look terrible.
![Tailscale  Improve UI for disconnecting exit node #6241 - Arc - 2023-05-01 at 11 43 48@2x](https://user-images.githubusercontent.com/13333578/235480534-3a60bc52-c30a-4f76-86d6-9404a27c2828.png)